systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaConnectorUtils.java

```diff
@@ -110,31 +118,45 @@ public static void waitForConnectorsTaskMaxChange(String connectorName, int task
         );
     }
 
+    public static String getConnectorSpecFromConnectAPI(String namespaceName, String podName, String connectorName) {
+        return cmdKubeClient(namespaceName).execInPod(podName, "/bin/bash", "-c",
+            "curl http://localhost:8083/connectors/" + connectorName).out();
+    }
+
     public static String getConnectorSpecFromConnectAPI(String podName, String connectorName) {
         return cmdKubeClient().execInPod(podName, "/bin/bash", "-c",
             "curl http://localhost:8083/connectors/" + connectorName).out();
     }
 
+    public static String getConnectorConfig(String namespaceName, String podName, String connectorName, String apiUrl) {
```

Comment:
Looks like you don't use `namespaceName`

Reply:
Good catch :))